gatsby/content/projects/bridges/matrix-bifrost.mdx

```diff
@@ -3,18 +3,29 @@ layout: projectimage
 title: matrix-bifrost
 categories: 
  - bridge
-description: bridging with a variety of backends including libpurple and xmpp.js
+description: bridging with xmpp.js
 author: Half-Shot
 maturity: Beta
 language: TypeScript
 license: Apache-2.0
 repo: https://github.com/matrix-org/matrix-bifrost
 room: "#bifrost:half-shot.uk"
 featured: true
-bridges: libpurple
-thumbnail: /images/libpurple.png
+bridges: XMPP
```

Comment:
Hm, can it be under multiple bridges. The bridge technically supports libpurple still, but XMPP should probably be considered the primary tech?

Reply:
The question is which is the target group you want to reach?
When users, then i would use XMPP; when developers then perhaps libpurple
I can say that a [part of the XMPP Community is upset that the XMPP Bridge isn't listed as XMPP.](https://www.freie-messenger.de/sys_matrix/gedanken/#datenintegrit%C3%A4t)